Xamarin.Forms.Core/BaseExportRendererAttribute.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+
+namespace Xamarin.Forms.Core
+{
+	[AttributeUsage(AttributeTargets.Assembly, AllowMultiple = true)]
+	public abstract class BaseExportRendererAttribute : HandlerAttribute
+	{
+		// not exposed publicly
+		protected abstract int? MajorVersion { get; }
+
+		#region MinimumSdkVersion
+		int _minimumSdkVersion = int.MinValue;
+		public int MinimumSdkVersion
+		{
+			get { return _minimumSdkVersion; }
+			set
+			{
+				if (_minimumSdkVersion == value)
+					return;
+
+				_minimumSdkVersion = value;
+
+				if (_minimumSdkVersion > MaximumSdkVersion)
+					throw new ArgumentException("Minimum SDK version must be less than or equal to the maximum SDK version.");
+			}
+		}
+		#endregion
+
+		#region MaximumSdkVersion
+		int _maximumSdkVersion = int.MaxValue;
+		public int MaximumSdkVersion
```

Comment:
I wonder if we should drop the SDK from here, @jassmith ?

Reply:
We should. Since this was a rewrite of an earlier PR, I did not pay attention to this detail. How about system version instead of sdk version? or api version?

Reply:
Since "version" means different things on different platforms (OS version, SDK verison, API version), I think we should just go with "version. So `MaximumVersion` and `MinimumVersion`.